### PR TITLE
Update ediscovery-diagnostic-info.md

### PR DESCRIPTION
**Pull request by Sokarpi**
this is long overdue . need to have the job diagnostics collection documented

### CODE REVIEW

File: microsoft-365/compliance/ediscovery-diagnostic-info.md [hunk 1]
```diff
@@ -76,6 +76,8 @@ Get-ComplianceCase "<eDiscovery (Standard) case name>"| %{$_|fl;"`t==Searches=="
 
 ## Collect diagnostic information for eDiscovery (Premium)
 
+Collect Case Information:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery-diagnostic-info.md]:
```suggestion
### Collect case information
```

File: microsoft-365/compliance/ediscovery-diagnostic-info.md [hunk 2]
```diff
@@ -76,6 +76,8 @@ Get-ComplianceCase "<eDiscovery (Standard) case name>"| %{$_|fl;"`t==Searches=="
...
+
 The **Settings** tab in an eDiscovery (Premium) case lets you quickly copy the diagnostic information for the case. The diagnostic information is saved to the clipboard so you can paste it to a text file and send to Microsoft Support.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery-diagnostic-info.md]:
```suggestion
The **Settings** tab in an eDiscovery (Premium) case lets you quickly copy the diagnostic information for the case. The diagnostic information is saved to the clipboard so you can paste it into a text file and send it to Microsoft Support.
```

File: microsoft-365/compliance/ediscovery-diagnostic-info.md [hunk 3]
```diff
@@ -88,6 +90,16 @@ The **Settings** tab in an eDiscovery (Premium) case lets you quickly copy the d
 
 5. Open a text file (in Notepad) and then paste the information in the text file.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery-diagnostic-info.md]:
```suggestion
5. Open a text file (in Notepad) and then paste the information into the text file.
```

File: microsoft-365/compliance/ediscovery-diagnostic-info.md [hunk 4]
```diff
@@ -88,6 +90,16 @@ The **Settings** tab in an eDiscovery (Premium) case lets you quickly copy the d
...
 
-6. Save the text file and name it something like `AeD Diagnostic Info YYYY.MM.DD` (for example, `AeD Diagnostic Info 2020.11.03`).
+6. Save the text file and name it something like `eDP Diagnostic Info YYYY.MM.DD` (for example, `eDP Diagnostic Info 2020.11.03`).
 
 After reviewing the file and redacting sensitive information, send it to the Microsoft Support engineer working on your case.
+
+Collect Job  Information:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery-diagnostic-info.md]:
```suggestion
### Collect job information
```

File: microsoft-365/compliance/ediscovery-diagnostic-info.md [hunk 5]
```diff
@@ -88,6 +90,16 @@ The **Settings** tab in an eDiscovery (Premium) case lets you quickly copy the d
...
+
+1. While in the Case, click the **Jobs** Tab.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery-diagnostic-info.md]:
```suggestion
1. While in the case, select the **Jobs** tab. 
```

File: microsoft-365/compliance/ediscovery-diagnostic-info.md [hunk 6]
```diff
@@ -88,6 +90,16 @@ The **Settings** tab in an eDiscovery (Premium) case lets you quickly copy the d
...
+6. Save the text file and name it something like `eDP Diagnostic Info YYYY.MM.DD` (for example, `eDP Diagnostic Info 2020.11.03`).
 
 After reviewing the file and redacting sensitive information, send it to the Microsoft Support engineer working on your case.
+
+Collect Job  Information:
+
+1. While in the Case, click the **Jobs** Tab. 
+2. Scroll down to the Job that faied and click on it to open the Job Information screen.
+3. Click the button **Copy Support Information** to copy the Job diagnostics information to the clipboard.
+4. Open a text file (in Notepad) and then paste the information in the text file.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery-diagnostic-info.md]:
```suggestion
2. Scroll down to the Job that failed and select it to open the **Job Information** screen.
3. Select **Copy Support Information** to copy the Job diagnostics information to the clipboard.
4. Open a text file (in Notepad) and then paste the information into the text file.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Sokarpi, kindly commit the suggested changes. Many thanks.

**Comment by Sokarpi** [comment]
Looks good.  Thank you!

Sonya Karpishpan | Office 365 Security & Compliance
Microsoft | Equivio
***@***.******@***.***> | +1 301 771 8687 | Maryland, USA
[https://outlook.office365.com/owa/service.svc/s/GetFileAttachment?id=AAMkAGQwY2MzOWVmLThjNTYtNDI3OC1hNzJjLTllYzAwYmQ2YmE4MQBGAAAAAABh7lj2WMxwTKbgos0IMgQlBwATgT%2BC2R5oR6FCa%2BUzCi%2FFAAAAhzHDAACrzQcSJghXTYgoDCCQcXYzAAAbZJQEAAABEgAQAP%2FjCZuH9%2BNOrqc8vHqL2Yo%3D&X-OWA-CANARY=fwtKD6QLBkGeppJbWJm-u3CkSs6k5dQYLgviJs_lYVIa0QY9y6ALjovYK7saDq7DMCKgFbSmo_k.&isImagePreview=True]

**Comment by yogkumgit** [comment]
@Sokarpi Please implement the suggested changes. Thanks

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by JohanFreelancer9** [comment]
@yogkumgit, my suggested changes have not yet been implemented.

**Comment by Sokarpi** [comment]
Thank you so much! It will save us a lot of time at support while collecting diagnostics.

Sonya Karpishpan | Office 365 Security & Compliance
Microsoft | Equivio
***@***.******@***.***> | +1 301 771 8687 | Maryland, USA
[https://outlook.office365.com/owa/service.svc/s/GetFileAttachment?id=AAMkAGQwY2MzOWVmLThjNTYtNDI3OC1hNzJjLTllYzAwYmQ2YmE4MQBGAAAAAABh7lj2WMxwTKbgos0IMgQlBwATgT%2BC2R5oR6FCa%2BUzCi%2FFAAAAhzHDAACrzQcSJghXTYgoDCCQcXYzAAAbZJQEAAABEgAQAP%2FjCZuH9%2BNOrqc8vHqL2Yo%3D&X-OWA-CANARY=fwtKD6QLBkGeppJbWJm-u3CkSs6k5dQYLgviJs_lYVIa0QY9y6ALjovYK7saDq7DMCKgFbSmo_k.&isImagePreview=True]

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by Sokarpi** [comment]
Sorry, I am not sure what else needs to be done here. It looks like the change was committed. Please let me know if I need to do anything else.

Thanks,

Sonya Karpishpan | M365 Security & Compliance
***@***.******@***.***> | +1 301 771 8687 | Maryland, USA

**Comment by JohanFreelancer9** [comment]
@Sokarpi, please click on every "Commit" button to commit each suggested change individually. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Review by Sokarpi — COMMENTED** [review]
approved

**Comment by Sokarpi** [comment]
I am a bit lost if this is all done, or what else needs to be done here to finish with this update, please ping me on Teams sokarpi@microsoft.com

**Review by Sokarpi — COMMENTED** [review]
good

**Review by Sokarpi — COMMENTED** [review]
no comments

**Comment by Sokarpi** [comment]
What else is needed to get this published? please advise

File: microsoft-365/compliance/ediscovery-diagnostic-info.md [hunk 7]
```diff
@@ -76,7 +76,9 @@ Get-ComplianceCase "<eDiscovery (Standard) case name>"| %{$_|fl;"`t==Searches=="
 
 ## Collect diagnostic information for eDiscovery (Premium)
 
-The **Settings** tab in an eDiscovery (Premium) case lets you quickly copy the diagnostic information for the case. The diagnostic information is saved to the clipboard so you can paste it to a text file and send to Microsoft Support.
+### Collect case information
+
+The **Settings** tab in an eDiscovery (Premium) case lets you quickly copy the diagnostic information for the case. The diagnostic information is saved to the clipboard so you can paste it into a text file and send it to Microsoft Support.
 
 1. Go to the compliance portal, and select **eDiscovery** > <a href="https://go.microsoft.com/fwlink/p/?linkid=2174006" target="_blank">**Advanced**</a>.
```
Comment (robmazz) [inline microsoft-365/compliance/ediscovery-diagnostic-info.md]:
```suggestion
1. Go to the compliance portal, and select **eDiscovery** > <a href="https://go.microsoft.com/fwlink/p/?linkid=2174006" target="_blank">**Premium**</a>.
```

File: microsoft-365/compliance/ediscovery-diagnostic-info.md [hunk 8]
```diff
@@ -86,8 +88,18 @@ The **Settings** tab in an eDiscovery (Premium) case lets you quickly copy the d
...
+5. Open a text file (in Notepad) and then paste the information into the text file.
 
-6. Save the text file and name it something like `AeD Diagnostic Info YYYY.MM.DD` (for example, `AeD Diagnostic Info 2020.11.03`).
+6. Save the text file and name it something like `eDP Diagnostic Info YYYY.MM.DD` (for example, `eDP Diagnostic Info 2020.11.03`).
 
 After reviewing the file and redacting sensitive information, send it to the Microsoft Support engineer working on your case.
+
+### Collect job information
+
+1. While in the case, select the **Jobs** tab.
```
Comment (robmazz) [inline microsoft-365/compliance/ediscovery-diagnostic-info.md]:
```suggestion
1. While in a case, select the **Jobs** tab. 
```

File: microsoft-365/compliance/ediscovery-diagnostic-info.md [hunk 9]
```diff
@@ -86,8 +88,18 @@ The **Settings** tab in an eDiscovery (Premium) case lets you quickly copy the d
...
 
-6. Save the text file and name it something like `AeD Diagnostic Info YYYY.MM.DD` (for example, `AeD Diagnostic Info 2020.11.03`).
+6. Save the text file and name it something like `eDP Diagnostic Info YYYY.MM.DD` (for example, `eDP Diagnostic Info 2020.11.03`).
 
 After reviewing the file and redacting sensitive information, send it to the Microsoft Support engineer working on your case.
+
+### Collect job information
+
+1. While in the case, select the **Jobs** tab. 
+2. Scroll down to the Job that failed and select it to open the **Job Information** screen.
```
Comment (robmazz) [inline microsoft-365/compliance/ediscovery-diagnostic-info.md]:
```suggestion
2. Scroll down to the Job that failed and select it to open the job information pane.
```

File: microsoft-365/compliance/ediscovery-diagnostic-info.md [hunk 10]
```diff
@@ -86,8 +88,18 @@ The **Settings** tab in an eDiscovery (Premium) case lets you quickly copy the d
...
+3. Select **Copy Support Information** to copy the Job diagnostics information to the clipboard.
```
Comment (robmazz) [inline microsoft-365/compliance/ediscovery-diagnostic-info.md]:
```suggestion
3. Select **Copy support information** to copy the job diagnostics information to the clipboard.
```

**Comment by robmazz** [comment]
@Sokarpi - thanks for the feedback. This is good information to clarify, incorporating with a separate PR. This PR never seems to validate the open license. Thanks again.

**Comment by Sokarpi** [comment]
Thank you Robert! Changes look good.

Sonya Karpishpan | M365 Security & Compliance
***@***.******@***.***> | +1 301 771 8687 | Maryland, USA

**Pull request closed without merging**